web/models/user.ex

```diff
@@ -6,6 +6,7 @@ defmodule Shareguru.User do
     field :email, :string
     field :picture, :string
     field :expire_at, Ecto.DateTime
+    field :hd, :string, virtual: true # Mail domain
```

Comment:
Hehehe, simplest solutions are best after all. Gotta say I didn't think about it XD